pkg/machines/xmlCreator.js

```diff
@@ -85,13 +85,25 @@ export function getNetworkXML({ name, forwardMode, device, ipv4, netmask, ipv6,
         forwardMode === 'open') {
         const domainElem = doc.createElement('domain');
         domainElem.setAttribute('name', name);
+        domainElem.setAttribute('localOnly', 'yes');
         networkElem.appendChild(domainElem);
     }
 
     if (ipv4) {
+        const dnsElem = doc.createElement('dns');
+        const hostElem = doc.createElement('host');
+        hostElem.setAttribute('ip', ipv4);
+        const hostnameElem = doc.createElement('hostname');
```

Comment:
Can someone explain this here? 
I usually use this xml syntax to assign static IPs to VMs with specific hostname, so where you put the gateway I usually put the VM's hostname? 
I am sure this here is something different, @skobyda or @vasile-papp if you can please clarify this, otherwise it's good to get merged.